Resources/Prototypes/Entities/Structures/Furniture/Tables/tables.yml

```diff
@@ -230,6 +233,9 @@
             max: 2
       - !type:DoActsBehavior
         acts: [ "Destruction" ]
+  - type: IconSmooth
+    key: metaltable
+    base: state_
```

Comment:
`base: state_` is written in TableBase so you can remove this line in all prototypes that inherits from TableBase